share/completions/emaint.fish

```diff
@@ -0,0 +1,32 @@
+## Global Opts
+complete -c emaint -s h -l help         -d "show this help message and exit"
+complete -c emaint -s c -l check        -d "Check for problems (a default option for most modules)"
+complete -c emaint      -l version      -d "show program's version number and exit"
+complete -c emaint -s f -l fix          -d "Attempt to fix problems (a default option for most modules)"
+complete -c emaint -s P -l purge        -d "Removes the list of previously failed merges. WARNING: Only use this option if you plan on manually fixing them or do not want them re-installed."
+
+## Subcommands
+complete -c emaint -n '__fish_use_subcommand' -xa 'all'         -d 'Perform all supported commands'
+complete -c emaint -n '__fish_use_subcommand' -xa 'binhost'     -d 'Scan and generate metadata indexes for binary packages.'
+complete -c emaint -n '__fish_use_subcommand' -xa 'cleanconfmem' -d 'Check and clean the config tracker list for uninstalled packages.'
+complete -c emaint -n '__fish_use_subcommand' -xa 'cleanresume' -d 'Discard emerge --resume merge lists'
+complete -c emaint -n '__fish_use_subcommand' -xa 'logs'        -d 'Check and clean old logs in the PORT_LOGDIR.'
```

Comment:
No trailing "." in the descriptions please.

Reply:
Done.